go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -331,6 +333,11 @@ func (c *addressManager) nodePrimaryAddrChanged() (bool, error) {
 
 // updateOVNEncapIP updates encap IP to OVS when the node primary IP changed.
 func (c *addressManager) updateOVNEncapIPAndReconnect() {
+	// Skip updating the encapsulation in the DPU mode, since it can have two instances
```

Comment:
This is here because we want to avoid DPU/FULL mode overriding each other?

Reply:
Right.
The short version is that the DPU and the HOST reference the same chassis inside the Southbound database. This means we have to be careful on ovn-encap-ip and hostname `external_ids`